skimage/measure/_marching_cubes.py

```diff
@@ -130,9 +142,6 @@ def marching_cubes(volume, level, spacing=(1., 1., 1.),
     verts = np.asarray(verts)
     faces = np.asarray(faces)
 
-    # Calculate gradient of `volume`, then interpolate to vertices in `verts`
-    grad_x, grad_y, grad_z = np.gradient(volume)
-
```

Comment:
We actually tried it but it wasn't satisfactory. The issue was only partially Cython. I believe the biggest problems were with the test suite, combined with the fact that PEP8 recommends grouping terms in mathematical equations naturally instead of blindly adding spaces. There are a lot of places where something like the first line below got flagged

``` python
out = img*x + 2*y - 3*x*z          # Readable, PEP8 correct, flagged by pyflakes
out = img * x + 2 * y - 3 * x * z  # Unreadable, PEP8 incorrect, passes pyflakes
```

despite being correct and preferred by PEP8, because nobody's implemented a flake8 checker that cares about order of operations.